gen/enum_test.go

```diff
@@ -508,19 +508,19 @@ func TestEnumLabelInvalidUnmarshal(t *testing.T) {
 	}{
 		{
 			`"some-random-str"`,
-			`unknown enum value "some-random-str" for "EnumWithLabel"`,
+			`unknown enum value "some-random-str" for "EnumWithLabel": strconv.ParseInt: parsing "some-random-str": invalid syntax`,
```

Comment:
optional: you can simplify this test by making it an `assert.Contains(t, err.Error(), msg)` check instead of `assert.Equal`. This will let you drop the part of the error message that's not under your control: the strconv.ParseInt failure.